### PR TITLE
Metric actions v 2

### PR DESCRIPTION
**Pull request by bCatanant**
Add new actions tab to metrics group view.

### CODE REVIEW

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  const actionType = screen.getByText('Circle promotion');
+  const triggeredAt = screen.getByText('08/10/2015 • 12:35:00');
+  const dropdown = screen.getByTestId('icon-vertical-dots');
+
+  expect(status).toBeInTheDocument();
+  expect(nickname).toBeInTheDocument();
+  expect(actionType).toBeInTheDocument();
+  expect(triggeredAt).toBeInTheDocument();
+
+  fireEvent.click(dropdown);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:43]:
```suggestion
  userEvent.click(dropdown);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('Edit action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:44]:
```suggestion
  userEvent.click(screen.getByText('Edit action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+  fireEvent.click(dropdown);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:46]:
```suggestion
  userEvent.click(dropdown);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('Delete action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:47]:
```suggestion
  userEvent.click(screen.getByText('Delete action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  render(
+    <ActionCard
+      handleDeleteAction={deleteActionFn}
+      handleEditAction={jest.fn()}
+      action={actionData}
+    />
+  );
+
+  const dropdown = screen.getByTestId('icon-vertical-dots');
+  fireEvent.click(dropdown);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:64]:
```suggestion
  userEvent.click(dropdown);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('Delete action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:65]:
```suggestion
  userEvent.click(screen.getByText('Delete action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('Yes, delete'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:66]:
```suggestion
  userEvent.click(screen.getByText('Yes, delete'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+  render(
+    <ActionCard
+      handleDeleteAction={deleteActionFn}
+      handleEditAction={jest.fn()}
+      action={actionData}
+    />
+  );
+
+  const dropdown = screen.getByTestId('icon-vertical-dots');
+  fireEvent.click(dropdown);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:82]:
```suggestion
  userEvent.click(dropdown);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('Delete action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:83]:
```suggestion
  userEvent.click(screen.getByText('Delete action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+  fireEvent.click(screen.getByText('No'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:84]:
```suggestion
  userEvent.click(screen.getByText('No'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/index.tsx [hunk 11]
```diff
@@ -76,80 +86,129 @@ const MetricsGroupCard = ({
...
+          ) : (
+            <NavTabs.Placeholder
+              title="No metric in this  group."
+              subTitle="You can add metrics and monitor the health of your activities."
+            />
+          )}
+          <Styled.MetricsGroupsFooter>
+            <Button.Default
+              onClick={() => handleAddMetric(metricGroup)}
+              id={'add-metric'}
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/index.tsx:187]:
```suggestion
              id="add-metric"
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/index.tsx [hunk 12]
```diff
@@ -76,80 +86,129 @@ const MetricsGroupCard = ({
...
-                handleEditMetric={handleEditMetric}
-              />
-            ))}
-          </Styled.MetricsGroupsCardContent>
-        </>
-      )}
+          <Styled.MetricsGroupsFooter>
+            <Button.Default
+              onClick={() => handleAddAction(metricGroup)}
+              id={'add-action'}
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/index.tsx:205]:
```suggestion
              id="add-action"
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 13]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and edit a metrics group', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupWithoutMetricData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const metricsGroupMenu = screen.getByTestId('icon-vertical-dots');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:161]:
```suggestion
  const metricsGroupMenu = await waitForElement(() => screen.queryByTestId('icon-vertical-dots'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 14]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  fireEvent.click(metricsGroupMenu);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:163]:
```suggestion
  userEvent.click(metricsGroupMenu);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 15]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+  fireEvent.click(screen.getByText('Edit'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:164]:
```suggestion
  userEvent.click(screen.getByText('Edit'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 16]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  const submit = screen.getByTestId('button-default-save');
+
+  expect(screen.queryByText('Edit metrics group')).toBeInTheDocument();
+  
+  fireEvent.click(submit);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:170]:
```suggestion
  userEvent.click(submit);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 17]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupWithoutMetricData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const addMetric = screen.getByTestId('button-default-add-metric');
+  
+  fireEvent.click(addMetric);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:184]:
```suggestion
  userEvent.click(addMetric);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 18]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupWithoutMetricData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
+
+  fireEvent.click(actionTab);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:202]:
```suggestion
  userEvent.click(actionTab);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 19]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
+
+  fireEvent.click(actionTab);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:277]:
```suggestion
  userEvent.click(actionTab);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 20]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
+
+  fireEvent.click(metricDropdown[1]);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:281]:
```suggestion
  userEvent.click(metricDropdown[1]);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 21]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+  fireEvent.click(screen.getByText('Edit action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:282]:
```suggestion
  userEvent.click(screen.getByText('Edit action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 22]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  const addActionForm = screen.getByTestId('metric-group-action-form');
+  const goBack = screen.getByTestId('icon-arrow-left');
+  
+  expect(addActionForm).toBeInTheDocument();
+
+  fireEvent.click(goBack);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:289]:
```suggestion
  userEvent.click(goBack);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 23]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
+
+  fireEvent.click(actionTab);
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
+
+  fireEvent.click(metricDropdown[1]);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:262]:
```suggestion
  userEvent.click(metricDropdown[1]);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 24]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+  fireEvent.click(screen.getByText('Delete action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:263]:
```suggestion
  userEvent.click(screen.getByText('Delete action'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 25]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
+
+  fireEvent.click(actionTab);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:258]:
```suggestion
  userEvent.click(actionTab);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 26]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
+
+  fireEvent.click(metricDropdown[1]);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:239]:
```suggestion
  userEvent.click(metricDropdown[1]);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 27]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+  fireEvent.click(screen.getByText('Edit metric'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:240]:
```suggestion
  userEvent.click(screen.getByText('Edit metric'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 28]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
+
+  fireEvent.click(metricDropdown[1]);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:224]:
```suggestion
  userEvent.click(metricDropdown[1]);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 29]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+  fireEvent.click(screen.getByText('Delete'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:225]:
```suggestion
  userEvent.click(screen.getByText('Delete'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 30]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
+
+  fireEvent.click(actionTab);
+
+  const addAction = screen.getByTestId('button-default-add-action');
+  
+  fireEvent.click(addAction);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:206]:
```suggestion
  userEvent.click(addAction);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 31]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and open new metric form', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupWithoutMetricData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const addMetric = screen.getByTestId('button-default-add-metric');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:182]:
```suggestion
  const addMetric = await waitForElement(() => screen.queryByTestId('button-default-add-metric'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 32]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and open new action form', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupWithoutMetricData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:200]:
```suggestion
  const actionTab = await waitForElement(() => screen.getByTestId('tab-1'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 33]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and delete metric', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:222]:
```suggestion
  const metricDropdown = await waitForElement(() => screen.getAllByTestId('icon-vertical-dots'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 34]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and edit metric', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const metricDropdown = screen.getAllByTestId('icon-vertical-dots');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:237]:
```suggestion
  const metricDropdown = await waitForElement(() => screen.getAllByTestId('icon-vertical-dots'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 35]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and delete action', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:256]:
```suggestion
  const actionTab = await waitForElement(() => screen.getByTestId(tab-1'));
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx [hunk 36]
```diff
@@ -148,3 +148,143 @@ test('render metrics groups and delete a metrics group', async () => {
...
+test('render metrics groups and edit action', async () => {
+  (fetch as FetchMock).mockResponseOnce(
+    JSON.stringify(metricsGroupData)
+  );
+
+  render(<MetricsGroups id={'1'} onGoBack={() => { }}/>);
+
+  await wait();
+
+  const actionTab = screen.getByTestId('tab-1');
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/__tests__/MetricsGroups.spec.tsx:275]:
```suggestion
  const actionTab = await waitForElement(() => screen.getByTestId(tab-1'));
```

**Review by ifdouglas — CHANGES_REQUESTED** [review]

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 37]
```diff
@@ -0,0 +1,101 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen, fireEvent, wait } from 'unit-test/testUtils';
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:19]: `wait` is not been used here

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 38]
```diff
@@ -0,0 +1,101 @@
...
+  const actionType = screen.getByText('Circle promotion');
+  const triggeredAt = screen.getByText('08/10/2015 • 12:35:00');
+  const dropdown = screen.getByTestId('icon-vertical-dots');
+
+  expect(status).toBeInTheDocument();
+  expect(nickname).toBeInTheDocument();
+  expect(actionType).toBeInTheDocument();
+  expect(triggeredAt).toBeInTheDocument();
+
+  fireEvent.click(dropdown);
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:44]:
```suggestion
  userEvent.click(dropdown);
```

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx [hunk 39]
```diff
@@ -0,0 +1,101 @@
...
+  fireEvent.click(screen.getByText('Edit action'));
```
Comment (ifdouglas) [inline ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:45]:
```suggestion
  userEvent.click(screen.getByText('Edit action'));
```

**Review by ifdouglas — CHANGES_REQUESTED** [review]

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx (hunk 38 above)
Comment (washingtonsoares) [reply ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:44]: It only works with `fireEvent`

File: ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx (hunk 39 above)
Comment (washingtonsoares) [reply ui/src/modules/Circles/Comparation/Item/MetricsGroups/MetricsGroupCard/__tests__/ActionCard.spec.tsx:45]: It only works with `fireEvent`

**Review by ifdouglas — APPROVED** [review]

**Review by washingtonsoares — APPROVED** [review]

**Review by ifdouglas — APPROVED** [review]

**Pull request merged**